src/lib.rs

```diff
@@ -3,6 +3,22 @@
 //! DBC files are descriptions of CAN frames.
 //! See [this post](https://www.kvaser.com/developer-blog/an-introduction-j1939-and-dbc-files/)
 //! for an introduction.
+//!
+//! # Usage
+//!
+//! ```
```

Comment:
this is run as a doctest, right? does it find the `foo.dbc` file? maybe have a hidden like like `# let dbc_path = "../testing/dbc-examples/example.dbc";` above it and use the var. people would stumble on that anyway and can't just copy the example code with our path anyway